custom/_legacy/hsph/tasks.py

```diff
@@ -95,10 +92,8 @@ def new_update_case_properties():
     past_42_date = past_x_date(time_zone, 42)
     setup_indices()
     for domain in DOMAINS:
-        case_list = list(get_cases_in_domain(domain, type=BIRTH_TYPE))
-        case_list = case_list + list(get_cases_in_domain(domain, type=CATI_FIDA_CHECK_TYPE))
-        cases_to_modify = []
-        for case in case_list:
+        for case in chain(get_cases_in_domain(domain, type=BIRTH_TYPE),
+                          get_cases_in_domain(domain, type=CATI_FIDA_CHECK_TYPE)):
```

Comment:
No point in fetching the whole case if you're just returning the ID. Use `get_case_ids_in_domain` instead